compiler/plugins/target/LLVMCPU/LLVMTargetOptions.cpp

```diff
@@ -581,6 +582,11 @@ void LLVMCPUTargetCLOptions::bindOptions(OptionsBinder &binder) {
                        targetVectorWidthInBytes, llvm::cl::cat(category),
                        llvm::cl::desc("Overrides the native vector register "
                                       "width (in bytes) of the target."));
+  binder.opt<unsigned>(
```

Comment:
you can use `llvm::cl::ByteSize` - then you can pass just a number like this or things like `64kib` - or `llvm::cl::PowerOf2ByteSize` if you want to enforce power-of-two values (include `iree/compiler/Utils/OptionUtils.h`)

Reply:
Thanks, applied! Would it make sense to add a negative LIT check for non-pow-2 inputs?